recipes/libcurl/all/conanfile.py

```diff
@@ -246,6 +255,9 @@ def _get_configure_command_args(self):
 
         params.append("--with-brotli" if self.options.with_brotli else "--without-brotli")
 
+        if tools.Version(self.version) >= "7.72.0":
+            params.append("--with-zstd" if self.options.with_zstd else "--without-zstd")
```

Comment:
What about zstd option in CMake build?

Reply:
https://github.com/conan-io/conan-center-index/blob/353ac6b1d1c2cd878b6e2903e0460ab54125a75b/recipes/libcurl/all/conanfile.py#L467 should have something added

Reply:
https://github.com/curl/curl/blob/e6b21d422e631a7c0cc81abf956af179b3b4c5e8/CMakeLists.txt#L661-L675
A `CURL_ZSTD` definition.

The name of the zstd cmake find_package is custom.
Zstd vs 
https://github.com/conan-io/conan-center-index/blob/4ac63884d528d6abb2d4740acb88d5cbd3d6b76a/recipes/zstd/all/conanfile.py#L70

Reply:
Yes, zstd only exports its targets since 1.4.5, so it makes sense for libcurl to use a custom module file.